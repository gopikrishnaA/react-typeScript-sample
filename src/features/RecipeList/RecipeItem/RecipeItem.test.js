import React from "react";
import { shallow } from "enzyme";
import renderer from 'react-test-renderer';
import { RecipeItem } from './RecipeItem';


describe("RecipeItem", () => {
    let wrapper;
    const recipe = {
        id: 123,
        title: 'test',
        readyInMinutes: '123'
    }
    beforeEach(() => {
        wrapper = shallow(<RecipeItem recipe={recipe} />);
    })

    it("renders correctly", () => {
        const tree = renderer
        .create(wrapper.find('.recipe-item'))
        .toJSON();
      expect(tree).toMatchSnapshot();
    })
})