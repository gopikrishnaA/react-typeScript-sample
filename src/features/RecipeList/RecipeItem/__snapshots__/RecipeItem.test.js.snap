// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeItem renders correctly 1`] = `
<div
  className="recipe-item p-2  d-flex align-items-start justify-content-start flex-column flex-lg-row "
>
  <img
    alt="test"
    height="40"
    src="https://spoonacular.com/recipeImages/123-90x90.jpg"
  />
  <div>
    test
    <span>
      123
       min
    </span>
  </div>
</div>
`;
